src/config.ts

```diff
@@ -49,6 +49,15 @@ export function getMergedConfig(): ConsentManagerConfig {
       ? jsonParseSafe(settings.consentManagerConfig, () => ({}))
       : settings.consentManagerConfig || {};
 
+  const settingsConfigInitialViewStateByPrivacyRegime =
+    settingsConfig?.initialViewStateByPrivacyRegime;
+  // Skip initialViewStateByPrivacyRegime config in settings if empty
+  if (
```

Comment:
i see whats going on, maybe we just fix this on the backend?

Reply:
https://github.com/transcend-io/consent-manager-ui/pull/50/files

Reply:
Yeah ideally we should fix it there. It can't hurt to have extra failsafes in the UI module though

Reply:
you know, it doesnt hurt to keep this around, just some defensive code